extensions/warp-ipfs/src/store/message.rs

```diff
@@ -2294,7 +2294,7 @@ impl ConversationTask {
             let mut streams: SelectAll<_> = SelectAll::new();
 
             for file in files {
-                match file {
+                match file.clone() {
```

Comment:
You can borrow this instead

```suggestion
                match &file {
```